views/js/test/runner/plugins/probes/latency/test.js

```diff
@@ -229,7 +231,9 @@ define([
         var testRunner = eventifier({
             getProxy: function() {
                 return {
-                    sendVariables: function() {}
+                    sendVariables: function() {
+                        return new Promise();
```

Comment:
Why returning an endless promise?
At least, please add a timeout and make the promise resolve at some point.

Reply:
fixed